proto/warden/async/v1beta1/query.proto

```
@@ -0,0 +1,80 @@
+syntax = "proto3";
+package warden.async.v1beta1;
+
+import "amino/amino.proto";
```

buf (1.47.2)

import "amino/amino.proto": file does not exist

(COMPILE)